apps/web/src/server/applications/create/key-dates/applications-create-key-dates.controller.js

```diff
@@ -0,0 +1,74 @@
+import * as applicationsCreateCaseService from '../case/applications-create-case.service.js';
+
+/** @typedef {import('./applications-create-key-dates.types').ApplicationsCreateKeyDatesProps} ApplicationsCreateKeyDatesProps */
+/** @typedef {import('./applications-create-key-dates.types').ApplicationsCreateKeyDatesBody} ApplicationsCreateKeyDatesBody */
+
+/**
+ * View the key dates step of the application creation
+ *
+ * @type {import('@pins/express').RenderHandler<ApplicationsCreateKeyDatesProps,
+ * {}, {}, {}, {}>}
+ */
+export async function viewApplicationsCreateKeyDates(req, response) {
+	const { applicationId } = response.locals;
+	const { keyDates } = await applicationsCreateCaseService.getApplicationDraft(applicationId);
+	const { firstNotifiedDate, submissionDate } = keyDates || {};
+
+	const values = {
+		'keyDates.firstNotifiedDate': firstNotifiedDate,
+		'keyDates.submissionDate': submissionDate
+	};
+
+	return response.render('applications/create/key-dates/_key-dates', { values });
+}
+
+/**
+ * View the key dates step of the application creation
+ *
+ * @type {import('@pins/express').RenderHandler<ApplicationsCreateKeyDatesProps,
+ * {}, ApplicationsCreateKeyDatesBody, {}, {}>}
+ */
+export async function updateApplicationsCreateKeyDates(
+	{ body, errors: validationErrors },
+	response
+) {
+	const { applicationId } = response.locals;
+	const {
+		firstNotifiedDay,
+		firstNotifiedMonth,
+		firstNotifiedYear,
+		submissionDay,
+		submissionMonth,
+		submissionYear
+	} = body;
+
+	const firstNotifiedSeconds =
+		(Date.parse(`${firstNotifiedYear}-${firstNotifiedMonth}-${firstNotifiedDay}`) || 0) / 1000;
```

Comment:
Why not sure here `new Date(firstNotifiedYear, firstNotifiedMonth, firstNotifiedDay)`?